docs/list.html

```diff
@@ -0,0 +1,428 @@
+                        </li>
+                       </a>
+                       <a href="../docs/modal.html" class="text-deocration-none selected-text">
+                        <li class="sidebar-text flex flex-align-item-center gap-1">
+                            <box-icon name='card' color='#eeeded' ></box-icon>
+                            <span>Modal</span>
+                        </li>
+                       </a>
+                       <a href="../docs/navigation.html" class="text-deocration-none selected-text">
+                        <li class="sidebar-text flex flex-align-item-center gap-1">
+                            <box-icon name='navigation' color='#eeeded' ></box-icon>
+                            <span>Navbars</span>
+                        </li>
+                       </a>
+                       <a href="../docs/rating.html" class="text-deocration-none selected-text">
+                        <li class="sidebar-text flex flex-align-item-center gap-1">
+                            <box-icon name='star-half' type='solid' color='#eeeded' ></box-icon>
+                            <span>Ratings</span>
+                        </li>
+                       </a>
+                       <a href="../docs/toast.html" class="text-deocration-none selected-text">
+                        <li class="sidebar-text flex flex-align-item-center gap-1">
+                            <box-icon name='copy' color='#eeeded' ></box-icon>
+                            <span>Toasts</span>
+                        </li>
+                       </a>
+                       <a href="../docs/typography.html" class="text-deocration-none selected-text">
+                        <li class="sidebar-text flex flex-align-item-center gap-1">
+                            <box-icon name='font-family' color='#eeeded' ></box-icon>
+                            <span>Typography</span>
+                        </li>
+                       </a>
+                    </ul>
+                        
+                 </div>
+            </div>
+             <div class="flex mt-1 ml-1 flex-align-item-center">
+                <svg width="40" height="32" viewBox="0 0 40 32" aria-hidden="true">
+                    <mask id="32e25648-9cb3-4aba-a168-64581e7aa254" width="52" height="52">
+                        <circle cx="16" cy="16" r="16" fill="white"></circle>
+                        <rect color="black" x="19" y="19" width="16" height="16" rx="8" ry="8"></rect>
+                    </mask>
+                    <foreignObject x="0" y="0" width="32" height="32" mask="url(#32e25648-9cb3-4aba-a168-64581e7aa254)">
+                        <div class="avatarStack-2Dr8S9"><img src="https://cdn.discordapp.com/avatars/709733291946672128/63f8ccaab30e072ae3a184ab142ddc54.webp" alt=" " class="avatar-user-icon" aria-hidden="true"></div>
+                    </foreignObject><svg x="14.5" y="17" width="25" height="15" viewBox="0 0 25 15">
```

Comment:
Hey here alt tag is empty please provide some relevant name to the image or component.